funcx_web_service/__init__.py

```diff
@@ -62,6 +65,57 @@ def create_tables():
         db.init_app(application)
         db.create_all()
 
+    # this is called before every request
+    @application.before_request
+    def before_request():
+        # this is basic request data
+        extra = {
+            "request_json": request.json,
+            "path": request.path,
+            "full_path": request.full_path,
+            "method": request.method,
+            "type": "before_request"
+        }
+        # this is additional data passed into the request URL via
+        # the view arguments (e.g. in '/tasks/<task_id>', the value of
+        # task_id is a view argument, so this will make task_id a
+        # key in the logged JSON object)
+        if request.view_args:
+            extra.update(request.view_args)
```

Comment:
I'd be inclined to put `view_args` as it's own nested dict in `extra`.  If we eventually add an endpoint that has a path "/do_something_to_type/<str:type>/other_stuff" this will clobber values already in the log.  It would probably be better to just assign view_args to it's own key.

Reply:
@joshbryan-globus Just fixed this, I missed it yesterday. I also renamed the `type` key to `log_type` just to be more specific and try to completely avoid such a situation